### PR TITLE
new package: wget2

### PR DESCRIPTION
**Pull request by shoya0x00**
#6167

### CODE REVIEW

File: packages/wget2/build.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+TERMUX_PKG_MAINTAINER="Rabby Sheikh @shoya0x00"
+TERMUX_PKG_VERSION=1.99.2
+TERMUX_PKG_SRCURL=https://mirrors.kernel.org/gnu/wget/wget2-${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=cbc48f55fa22ed2acbccf032c208c133cc59c7432cda8518a4992eb5882b6563
+TERMUX_PKG_DEPENDS="openssl, libandroid-glob, libgpg-error, libassuan, gpgme"
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="ac_cv_header_spawn_h=no --with-ssl=openssl"
+
+termux_step_pre_configure() {
+	LDFLAGS+=" -landroid-glob"
+    CFLAGS+=" -DNO_INLINE_GETPASS=1"
```
Comment (Grimler91) [inline packages/wget2/build.sh:13]: Please fix the indentation, use tabs only

**Comment by shoya0x00** [comment]
When I Build With My Device Everything Works Fine. But When I Use The Docker Builded Package It Gives This Error. And I Don't Know Which File Is Causing This 😑
```
~ $ wget2 http://192.168.0.102:8000/wget2_1.99.2_aarch64.deb
Unknown configuration key 22 (maybe this config value should be of another type?)
Unknown configuration key 21 (maybe this config value should be of another type?)
[0] Downloading 'http://192.168.0.102:8000/wget2_1.99.2_aarch64.deb' ...
Saving 'wget2_1.99.2_aarch64.deb.1'
HTTP response 200 OK [http://192.168.0.102:8000/wget2_1.99.2_aarch64.deb]
```

**Comment by xeffyr** [comment]
> But When I Use The Docker Builded Package It Gives This Error

Sometimes Answers Can Be Found On Upstream, https://gitlab.com/gnuwget/wget2/-/issues/537.

**Comment by shoya0x00** [comment]
> > But When I Use The Docker Builded Package It Gives This Error
> 
> Sometimes Answers Can Be Found On Upstream, https://gitlab.com/gnuwget/wget2/-/issues/537.
> 
> `openssl` causes this.

Never checked the upstream 😑😑😑

**Pull request merged**